doc-content/jbpm-docs/src/main/asciidoc/ElasticSearch/integration-elasticsearch-proc.adoc

```diff
@@ -0,0 +1,42 @@
+[id='integration-elasticsearch-proc_{context}']
+= Configuring the {KIE_SERVER} to send information to ElasticSearch when a transaction is committed
+
+You can configure the {KIE_SERVER} to send information to ElasticSearch about every event when a process, case, or task is completed. The {KIE_SERVER} sends the information when it commits transactions.
+
+You can use this functionality with any business process or case. You do not need to change anything in the process design.
+
+This configuration is also available if you run your process service using SpringBoot.
+
+The {KIE_SERVER} serializes the process, case, and task information as JSON documents.
+
+.Procedure
+
+. To enable sending information to ElasticSearch, complete one of the following steps:
+.. If you deployed the {KIE_SERVER} on {EAP} or another application server:
+ifdef::PAM,DM[]
+... Download the `{PRODUCT_FILE}-maven-repository.zip` product deliverable file from the {PRODUCT_DOWNLOAD_LINK}[Software Downloads] page of the Red Hat Customer Portal.
+... Extract the contents of the file.
+... Copy the `maven-repository/org/jbpm/jbpm-event-emitters-elasticsearch/{MAVEN_ARTIFACT_VERSION}/jbpm-event-emitters-kafka-{MAVEN_ARTIFACT_VERSION}.jar` file into the `deployments/kie-server.war/WEB-INF/lib` subdirectory of the application server.
+endif::PAM,DM[]
+ifdef::JBPM,DROOLS,OP[]
+... Retrieve the `org.jbpm.jbpm-event-emitters-elasticsearch` JAR file version `{MAVEN_ARTIFACT_VERSION}` from the public Maven repository.
+... Copy the file into the `deployments/kie-server.war/WEB-INF/lib` subdirectory of the application server.
+endif::JBPM,DROOLS,OP[]
++
+.. If you deployed the application using SpringBoot, add the following lines to the `<dependencies>` list in the `pom.xml` file of your service:
++
+[source,xml]
+----
+<dependency>
+  <groupId>org.jbpm</groupId>
+  <artifactId>jbpm-event-emitters-elasticsearch</artifactId>
+  <version>${version.org.kie}</version>
+</dependency>
+----
++
+. Configure any of the following system properties for the {KIE_SERVER} as necessary:
```

Comment:
Property names are prefixed with 
org.jbpm.event.emitters.elasticsearch
Please change accordingly.
PS: Since you are asking in PR, Index in elastic search is roughly equivalent to a relational db table